### PR TITLE
Fixes 291: Add iqe-tests to pr_check

### PR DESCRIPTION
**Pull request by swadeley**
HMSCONTENT-291

### CODE REVIEW

**Comment by jlsherrill** [comment]
I only see the dummy.dummytest (from (junit-dummy.xml))  test, should i see more with this?  (or do we have to merge it first)?

**Comment by jlsherrill** [comment]
@mshriver question for you^

**Comment by mshriver** [comment]
@jlsherrill yep! @swadeley check out the rest of the pr_check file - it has a block at the end that is a stop-gap for the fact that tests weren't actually being executed in the default bonfire cicd scripts.

File: .rhcicd/pr_check.sh [hunk 1]
```diff
@@ -22,22 +22,37 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
...
+export IQE_FILTER_EXPRESSION=""
+export IQE_ENV="ephemeral"
+export IQE_SELENIUM="true"
+export IQE_CJI_TIMEOUT="30m"
+export DEPLOY_TIMEOUT="900"  # 15min
+export DEPLOY_FRONTENDS="true"
+
+# bootstrap bonfire and it's config
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s "$CICD_URL"/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_boot
```
Comment (mshriver) [inline .rhcicd/pr_check.sh:36]: missing the end of the filename here, got cut off at `.cicd_boot`, so it never ran.

Comment (mshriver) [inline .rhcicd/pr_check.sh:36]:
```suggestion
curl -s "$CICD_URL"/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
```

File: .rhcicd/pr_check.sh [hunk 2]
```diff
@@ -22,22 +22,37 @@ COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-fronten
...
-</testsuite>
-EOF
-
-# teardown_docker
-exit $BUILD_RESULTS
+
+
+# reserve ephemeral namespace
+export DEPLOY_FRONTENDS="true"
+export APP_NAME="content_sources"
```
Comment (mshriver) [inline .rhcicd/pr_check.sh:45]: This doesn't need to change, it was set at the top as `content-sources`, which is correct.

**Review by mshriver — APPROVED** [review]
Navigation failure in one test, otherwise check is executing as expected.

**Comment by swadeley** [comment]
> Navigation failure in one test, otherwise check is executing as expected.

Its a Permission error in UI: "You do not have access to Repositories"

**Comment by mshriver** [comment]
The failure is not impacted by this PR's content, I'm going to merge to enable pr_checks to run the tests.

**Comment by swadeley** [comment]
Thank you

**Pull request merged**